tasks/5_profile-lookup/profile-lookup.js

```diff
@@ -0,0 +1,40 @@
+export const profiles = [
+  {
+    firstName: 'Kristian',
+    lastName: 'Vos',
+  },
+  {
+    firstName: 'Sherlock',
+    likes: [
+      'Intriguing Cases',
+      'Violin',
+    ],
+  },
+  {
+    firstName: 'Harry',
+    likes: [
+      'Pizza',
+      'Pasta',
+    ],
+  },
+  {
+    firstName: 'Akira',
+  },
+];
+
+
+export function lookupProfile(firstName, prop) {
+  let message = '';
+  for (let profile of profiles) {
+    if (profile.firstName === firstName) {
+      if (profile.hasOwnProperty(prop)) {
```

Comment:
you can merge this profile.hasOwnProperty(prop) and if (profile.firstName === firstName) - it will reduce complexity